test/integration/team-membership-integration.js

```diff
@@ -1,7 +1,7 @@
 export default function teamMembershipTests (t, organization) {
   t.test('Gets teamMemberships', (t) => {
```

Comment:
another nit: maybe describe this test as being "get team memberships for team" or something like that, and the one below could be "get team memberships for org"